app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for human in people:
+        Person(human["name"], human["age"])
+
+    for human in people:
+        if human.get("wife"):
+            Person.people[human["name"]].wife = Person.people[human["wife"]]
+        if human.get("husband"):
+            Person.people[human["name"]].husband \
+                = Person.people[human["husband"]]
```

Comment:
The line continuation character `\` at the end of line 17 is unnecessary here. Python automatically continues lines inside parentheses, brackets, and braces. Consider removing it for cleaner code.